include/declarations.h

```diff
@@ -15,6 +15,12 @@ extern void (*gUnknown_0811DFFC[6])(struct AnimationStruct *, struct CourtScroll
 extern u32 * gFlagPtrs[3]; 
 extern s16 gSineTable[256+64];
 extern u32 gUnknown_080150D0[12];
+extern const u16 gUnknown_08015100[0x78/2];
+extern const u16 gUnknown_08015178[0x64/2];
+extern const u16 gUnknown_080151DC[0xDC/2];
+extern const u16 gUnknown_080152B8[0x28/2];
+extern const u16 gUnknown_080152E0[0x64/2];
+extern const u16 gUnknown_08015344[0x78/2];
```

Comment:
These shouldn't be u16  as they are examination data and there already exists a struct for these in the code called `ExaminationData`